engine/src/main/java/org/hibernate/search/cfg/ParameterAnnotationsReader.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.cfg;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+import org.hibernate.search.annotations.Parameter;
+import org.hibernate.search.util.logging.impl.Log;
+import org.hibernate.search.util.logging.impl.LoggerFactory;
+
+/**
+ * Helper to convert the org.hibernate.search.annotations.Parameter
+ * annotations.
+ */
+public final class ParameterAnnotationsReader {
```

Comment:
You can, but that's one more OSGi problem, because this must be used in the Elasticsearch module :)